apps/condo/pages/playground/iframe.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import Head from 'next/head'
+import { PageWrapper } from '@condo/domains/common/components/containers/BaseLayout'
+import { IFrame } from '@condo/domains/common/components/IFrame'
+import { useOrganization } from '@core/next/organization'
+import get from 'lodash/get'
+
+const ContactsPage = () => {
+    const { organization } = useOrganization()
+
+    return (
+        <>
+            <Head>
+                <title>Страница с iframe</title>
+            </Head>
+            {/*<FeatureFlagRequired name={'playground'} fallback={<Error statusCode={404}/>}>*/}
```

Comment:
I think we should remove commented code from pr

Reply:
See todo above...I'll remove this page...this is just for playground-test